### PR TITLE
refactor: refactor readme and app.py for readability

### PR DESCRIPTION
**Pull request by TsubasaK111**
refactored README of both languages and app.py template to improve readability.

### CODE REVIEW

**Comment by TsubasaK111** [comment]
@shunjikonishi review me!

File: app/app.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
 def main(args, options):
-  for v in args:
-    print(v)
+  for arg in args:
+    result = do_something(arg)
```
Comment (shunjikonishi) [inline app/app.py]: do_something is not defined.

**Comment by shunjikonishi** [comment]
@TsubasaK111 fix me
app.py should work without editing.
As default, app.py works as an echo application.

**Comment by TsubasaK111** [comment]
@shunjikonishi As you requested, app.py now works without editing, and echoes back. Review me!

**Comment by shunjikonishi** [comment]
LGTM

**Pull request merged**